packages/panels/docs/14-upgrade-guide.md

```diff
@@ -178,7 +178,18 @@ A side effect of this change is that all custom icons that you use must now be [
 
 #### Logo customization
 
-In v2, you can customize the logo of the admin panel using a `/resources/views/vendor/filament/components/brand.blade.php` file. In v3, this has been moved to `/resources/views/vendor/filament-panels/components/logo.blade.php`.
+In v2, you can customize the logo of the admin panel using a `/resources/views/vendor/filament/components/brand.blade.php` file. In v3, this has been moved to `/resources/views/vendor/filament-panels/components/logo.blade.php`. Additionally a `brandLogo` API has been added. You can set the logo by adding `brandLogo()` to your panel configuration.
```

Comment:
I think we can remove the old part about overriding in v3.
Also, could you link to the `brandLogo()` docs instead of the inline example?

Reply:
I re-added the overriding part, since it's still part of the framework and added the second option with the brandLogo API.

I also added a link to the specify part of the docs for brandLogo.

Reply:
Yeah it still is a valid way, because we can't remove it in a non-breaking way.
However, we like to keep our documentation up-to-date with the latest approach, which makes it easier to migrate away from it and remove it in a breaking way in a future major version. Even though this is a tiny thing, I think it would be good to teach people about the new API only, since it's much cleaner and has a smaller likelihood of issues.

Reply:
I see. I removed the semi-deprecated way. :)